packages-exp/auth-exp/src/core/user/id_token_result.ts

```diff
@@ -84,6 +84,7 @@ export async function getIdTokenResult(
     )!,
     signInProvider: signInProvider || null,
     signInSecondFactor: firebase?.['sign_in_second_factor'] || null
+    // TODO(lisajian): Add estimatedLocalExpirationTime
```

Comment:
You can remove this TODO -- we opted not to add this field.

Reply:
Sick, done